CancerGov/_src/Scripts/NCI/UX/PageSpecific/Inner/Enhancements/directory-search-results.js

```diff
@@ -0,0 +1,105 @@
+const checkedItems = JSON.parse(sessionStorage.getItem('directorySearchList')) || [];
+
+const trackCheckboxes = () => {
+  const checkboxes = document.querySelectorAll(".directory-search-terms ~ div .checkbox input");
+  if (checkboxes.length > 1) {
+    checkboxes.forEach(checkbox => {
```

Comment:
First time around I tried `getNodeArray` but then I couldn't map over the array. I think it may have been because I had a bad selector returning an empty array :S. I'll update this.